javascript-forms/form-events.js

```diff
@@ -0,0 +1,27 @@
+const inp = document.querySelectorAll('input');
+const formSubmit = document.querySelector('form');
+
+const handleFocus = (event) => {
+  console.log(`${event.type} event fired`);
+  console.log(`event.target.name: ${event.target.name}`);
+};
+const handleBlur = (event) => {
+  console.log(`${event.type} event fired`);
+  console.log(`event.target.name: ${event.type}`);
+};
+const handleInput = (event) => {
+  console.log(`${event.target.name} event fired`);
+  console.log(`event.target.value: ${event.target.value}`);
+};
```

Comment:
Don't use arrow functions for named functions. Only callback functions. Named functions should be using the `function` keyword